src/css/components/buttons.css

```diff
@@ -0,0 +1,39 @@
+.btn {
+	display: inline-block;
+	line-height: 2.5rem;
+	font-size: 1.5rem;
+	padding: 0 1.25rem;
+	margin-bottom: 0.5rem;
```

Comment:
Everything is good and clean !
Suggestion : use 'px' instead of rems when value is less than 1rem

Reply:
Thanks Varun for the suggesstion, updated the code